services/biosamples.py

```diff
@@ -0,0 +1,74 @@
+from typing import List
+from biosamples_v4.api import Client as BioSamplesClient
+from biosamples_v4.aap import Client as AapClient
+from biosamples_v4.encoders import SampleEncoder
+from biosamples_v4.models import Attribute, Sample
+from excel.clean import object_has_attribute, value_populated
+
+
+def attribute_name(name: str) -> str:
+    return name.replace('_', ' ')
+
+
+def optional_attribute(sample: dict, attribute: str):
+    return sample[attribute] if object_has_attribute(sample, attribute) else None
+
+
+def make_attribute(name, value) -> Attribute:
+    units = None
+    # ToDo: Refactor to create the Sample at validation-time so the units are accessible from validation_map
+    if name in ['geographic_location_(latitude)', 'geographic_location_(longitude)']:
```

Comment:
These are `magic strings`. Is it better to put them into a config file and get them from there?

Reply:
Once the ToDo is implemented this won't be an issue.